console/console-init/ui/src/modules/address-space/utils/AddressSpaceFormatter.tsx

```diff
@@ -46,6 +48,9 @@ const statusToDisplay = (phase: string) => {
     case "pending":
       icon = <ExclamationCircleIcon color="red" />;
       break;
+    case "failed":
+      icon = <ErrorCircleOIcon color="red" />;
+      break;
     case "":
```

Comment:
do we need this blank case if yes then move it in default case. And always add default case in switch to avoid warning

Reply:
@ajaypratap003 Yes, I think we need the case for blank in phase as initially for few moment server returns empty phase and once controller picks up the address it will update the status as pending. So that's why there is a case for blank string